sdk/src/main/java/com/deploygate/sdk/DeployGate.java

```diff
@@ -53,8 +56,9 @@ public class DeployGate {
     private static final Object sPendingEventLock = new Object();
 
     private static final Object sLock = new Object();
-    private static CustomAttributes sBuildEnvironment = null;
-    private static CustomAttributes sRuntimeExtra = null;
+    private static CustomAttributes sBuildEnvironment;
+    private static CustomAttributes sRuntimeExtra;
+    private static CustomAttributes sSdkRuntimeExtra;
```

Comment:
This attrs will be used to collect data for valuable on the debugging.
It is always empty at this point.